components/filter-field/src/filter-field-tag/filter-field-tag.ts

```diff
@@ -66,9 +67,18 @@ export class DtFilterFieldTag {
     this._changeDetectorRef.markForCheck();
   }
 
+  /** Whether the parent filter field is disabled */
+  @Input()
+  get filterFieldDisabled(): boolean {
+    return this.data.filterFieldDisabled;
+  }
+  set filterFieldDisabled(value: boolean) {
+    this.data.filterFieldDisabled = coerceBooleanProperty(value);
+  }
+
```

Comment:
- That's not a good idea, because this would introduce a cyclic dependency. Tag should not know of the filter field.
- I am also sure why we need the `filterFieldDisabled` input and not just disable the whole tag via the `disabled` input? Store the current disabled values of the tags, set all of them to disabled and when you enable it again, apply the prev stored value.

Reply:
It allows the following workflow:
* Add 2 filters
* Disable the first tag
* Disable whole filter field
* Enable whole filter field
-> First tag stays disabled

Not sure though if this really is a use-case, I'm fine with only keeping a single disabled state.